### PR TITLE
Add energy thermostat api

### PR DESCRIPTION
**Pull request by robmelfi**
Homesdata retrieve user's homes and their topology  
Endpoint: https://api.netatmo.com/api/homesdata  
Referece: https://dev.netatmo.com/en-US/resources/technical/reference/energy/homesdata

Homestatus get the current status and data measured for all home devices  
Endpoint: https://api.netatmo.com/api/homestatus  
Reference: https://dev.netatmo.com/en-US/resources/technical/reference/energy/homestatus

### CODE REVIEW

**Comment by lost-carrier** [comment]
sry for the delay - was on vacation...

File: pom.xml [hunk 1]
```diff
@@ -32,7 +32,7 @@
     </scm>
 
 	<properties>
-		<java-version>1.7</java-version>
+		<java-version>1.8</java-version>
```
Comment (lost-carrier) [inline pom.xml]: As far as I can see there is not features of 1.8 used so I would leave it at 1.7 for better backward compatibility.
```suggestion
		<java-version>1.7</java-version>
```

File: pom.xml [hunk 2]
```diff
@@ -77,6 +77,17 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
```
Comment (lost-carrier) [inline pom.xml]: Hmm... not used? ...should be removed?

**Review by lost-carrier — CHANGES_REQUESTED** [review]
Could you add a small test in NetatmoHttpClientTest? (...afaik I cannot even see the response format  if I don't own such a thermostat, right?)

File: pom.xml (hunk 2 above)
Comment (robmelfi) [reply pom.xml]: yes

**Comment by robmelfi** [comment]
@lost-carrier I added the tests 👍🏼

**Comment by lost-carrier** [comment]
@robmelfi awesome - thanks!

**Pull request merged**